app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -109,7 +109,7 @@ enum class NavigationItem(
         primaryRoute = SELF_USER_PROFILE,
         canonicalRoute = "$SELF_USER_PROFILE/{$EXTRA_USER_ID}",
         content = { SelfUserProfileScreen() },
-        animationConfig = NavigationAnimationConfig.CustomAnimation(smoothSlideInFromRight(), smoothSlideOutFromLeft())
+        animationConfig = NavigationAnimationConfig.CustomAnimation(expandInToView(), shrinkOutFromView())
```

Comment:
Somehow it feels like only the initial `expandIn` animation is happening and not the `shrinkOut`. Could you identify why?

https://user-images.githubusercontent.com/2468164/161495542-d9cbd08c-264c-4bce-a34f-3990150f2995.mov

Reply:
I'm was having a look during the morning, and it seems there is no easy solution. We have 2 `NavGraphs` that are conflicting for the animation, the `HomeNavGraph` (the one with conversations) is taking over the Items `NavGraph`.

So when we are exiting out from `SelfUserProfileScreen` we are just seeing the Home/Conversations animation, that currently doesn't have.

I think, I could try some orchestration with the enter animation of the Home/Conversations as a last resort

Reply:
@gongracr Fixed it, I was able to achieve it by setting the animations on HomeScreen to null instead of None.
Since None was overriding the animation spec of the parent also, as stated in the documentatio https://google.github.io/accompanist/navigation-animation/